src/cli/utils/fixtures/asyn-load-config-test-fixture.ts

```diff
@@ -0,0 +1,67 @@
+export const asyncDataWithAsyncFunctionAndAsyncParameter = `		const axios=require('axios').default
```

Comment:
create one fixture file for each test config and pass the path to that file to the util method, also I dont think that axios is used no?

In the end, each fixture file is like a medusa-config-file
medusa-config-sync.js
medusa-config-async.js
medusa-config-sync-async.js

Reply:
it will depend on making the path configurable, as medusa's default behaviour is to only pick up medusa-config.js, I didn't want to alter that. 
wdyt ?

Reply:
In any case you can do it as it would just mean that the test need to cp the file in a different Location and the rm the file but that is fine

Reply:
done